.gitignore

```diff
@@ -8,3 +8,4 @@ public/polyfill.min.js
 static/*
 !static/info.txt
 test/frontend/bundle.js
+public/main.css
```

Comment:
I moved the public/main.css file to frontend/src/main.css. Then `npm run build:css` does the whole autoprefixer, media query combining, minification and source map magic and copies the _processed_ file back into the old public/main.css location.